### DISCUSSION TITLE
Major docs update: App Router-first

### DISCUSSION THREAD

**Discussion by amannn** [amannn/next-intl#351] [Announcements]
Hello everyone! 👋

With Next.js 13.4, the App Router was announced as stable. Following the lead of Next.js, next-intl now also recommends this paradigm going forward.

The [documentation site was rewritten accordingly](https://next-intl-docs-git-docs-app-router-first-next-intl.vercel.app/) and is ready for review ([PR](https://github.com/amannn/next-intl/pull/345)).

If you have any feedback, please let me know! Many thanks! 🙌 

**Update:** The new docs were published as of June 29th, but please keep the feedback coming!

**Comment by raphaelbadia** [comment]
Congrats on the release! Skimmed through the PR, looks good from a non-maintainer perspective 😅
